CHANGELOG

```diff
@@ -1,6 +1,8 @@
 2.8.0.dev (compared to 2.7.X)
 -----------------------------
 
+- merge the pytest-cache extension into core
```

Comment:
I think it's worth adding here how this will affect users with `pytest-cache` installed.

Also, add a "Thanks Ronny Pfannschmidt for the PR" note here. :wink: